### DISCUSSION TITLE
This project has been discontinued

### DISCUSSION THREAD

**Discussion by fernandopro** [htmlburger/carbon-fields#1229] [General]
It is important to inform the public that **this project has been discontinued**. There are numerous unresolved issues and defects in the latest versions of WordPress and PHP.

**Comment by ogorzalka** [comment]
Really ???

**Comment by hadamlenz** [comment]
wow that stinks

**Comment by JerschabekGmbH** [comment]
No I don´t think so. Important fixes are made from time to time.

**Comment by emohamed** [comment]
The project is not discontinued.

It is, however, feature-complete: the Gutenberg editor makes Carbon Fields unnecessary for the vast majority of CMS use-cases, so we see no point in building new features or pursue further use-cases for the library. Therefore, the repo is somewhat quiet. 

There are not major problems with the latest PHP or WordPress versions -- we maintain a lot of sites based on Carbon Fields and we resolve major issues when they arise.

At the end of the day, Carbon Fields is open-source project that we at htmlBurger maintain for our own needs. We might be slow in addressing deprecations and minor issues that might be important for you guys, however we do the best we can with the resources we have available.

**Reply by art3mis-pc** [reply]
_the Gutenberg editor makes Carbon Fields unnecessary for the vast majority of CMS use-cases_

I love the ease at which I can build custom blocks with Carbon Fields and use it on every single website I build. It will be a shame if that aspect of CF is not going to be updated further. It has a few things missing that would be really nice to have (proper support for block alignment toolbar, typography and spacing settings).

**Comment by hadamlenz** [comment]
With no updates in months it did seem plausible that you've abandoned the project.  That being said, making options pages was my primary use case, and this framework does it quite well.

There are a couple of fixes y'all need to do to get the code working well with WP 6.5
I see this one a lot `carbon-fields-core-css was added to the iframe incorrectly. Please use block.json or enqueue_block_assets to add styles to the iframe.` and `carbon-fields-blocks-css was added to the iframe incorrectly.` and #1206 also. 

I thought i saw the first two in the issue bin already.  I'll add an issue for them
edit : its #1230
